test/integration/helpers.sh

```diff
@@ -138,7 +138,7 @@ tpm2_test_original_cwd=""
 tpm2_test_cwd=""
 function switch_to_test_dir() {
     tpm2_test_original_cwd=`pwd`;
-    tpm2_test_cwd=$(mktemp --directory --tmpdir=/tmp tpm2_test_XXXXXX)
+    tpm2_test_cwd=$(mktemp -d /tmp/tpm2_test_XXXXXX)
```

Comment:
While you're at it, we should respect the [POSIX `$TMPDIR` environment variable](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html#tag_08_03) if it is set:
```
tpm2_test_cwd=$(mktemp -d "${TMPDIR:-/tmp}/tpm2_test_XXXXXX")